apps/backend/core/configuration/validation/validators/ai_validators.py

```
@@ -0,0 +1,166 @@
+        timeout: API timeout from config.timeouts.external_api_timeout
+        level: ValidationLevel.CRITICAL
+
+    Example:
+        >>> config = get_config()
+        >>> validator = OpenAIValidator(config)
+        >>> result = await validator.validate()
+        >>> if result.is_success:
+        ...     print(f"OpenAI API validated in {result.latency_ms:.1f}ms")
+    """
+
+    def __init__(self, config: BaseConfig):
```

Ruff (0.14.4)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)